src/views/orderReport.tsx

```diff
@@ -0,0 +1,467 @@
+import { ChartData } from "chart.js";
+import { Column } from "primereact/column";
+import { ColumnGroup } from 'primereact/columngroup';
+import { Row } from 'primereact/row';
+import { FC, useEffect, useState } from "react";
+import useScreenSize from "../hooks/useScreenSize";
+import LineChart from "../components/LineChart";
+import HorizontalTable from "../components/HorizontalTable";
+import CustomTab from "../components/common/customtab";
+import CustomTable from "../components/common/customtable";
+import Loader from "../components/loader";
+import OrderSummaryCard from "../components/OrderSummaryCard";
+import PieChart from "../components/PieChart";
+import { HOURLY_ORDER_TREND_LINE_CHART_OPTION, ORDER_REPORT_PIE_CHART_OPTIONS } from "../config/chartConfig";
+import { URL_HOURLYREPORT } from "../constants/apiConstants";
+import { CHANNELWISE_DATA_LABELS, HOURLY_TREND_DATA_LABELS, LOCALEWISE_DATA_LABELS, ORDER_PIE_CHART_LABELS, ORDER_REPORT_CHART_LABELS, ORDER_REPORT_LABELS, ORDER_REPORT_THRESHOLDS, ORDER_SUMMARY_LABELS, PAYMENTWISE_DATA_LABELS, REPORT_TABS, SCREEN_WIDTH } from "../constants/appConstants";
```

Comment:
This line should be in multiple lines if prettier is working as expected.
Can you check ?

Reply:
I had already compared my prettier config with Adithya and Sarath. We couldn't find any difference. This formatting is not working in my vs code